cmd/rebasehelpers/commitchecker/commitchecker.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift/origin/cmd/rebasehelpers/util"
+)
+
+func main() {
+	var start, end string
+	flag.StringVar(&start, "start", "master", "The start of the revision range for analysis")
+	flag.StringVar(&end, "end", "HEAD", "The end of the revision range for analysis")
+	flag.Parse()
+
+	commits, err := util.CommitsBetween(start, end)
+	if err != nil {
+		os.Stderr.WriteString(fmt.Sprintf("ERROR: couldn't find commits from %s..%s: %v\n", start, end, err))
+		os.Exit(1)
+	}
+
+	validators := []func([]util.Commit) error{
+		ValidateUpstreamCommitsWithoutGodepsChanges,
+		ValidateUpstreamCommitModifiesSingleGodepsRepo,
+		ValidateUpstreamCommitSummaries,
```

Comment:
Moved registration of validators into a central location which I now reference here.